pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -182,15 +182,15 @@ func (p *ConfigurationEditFlags) addSharedFlags(command *cobra.Command) {
 		"DEPRECATED: please use --limit instead. The limits on the requested memory (e.g., 1024Mi).")
 	p.markFlagMakesRevision("limits-memory")
 
-	command.Flags().IntVar(&p.MinScale, "min-scale", 0, "Minimal number of replicas.")
-	p.markFlagMakesRevision("min-scale")
-
-	command.Flags().IntVar(&p.MaxScale, "max-scale", 0, "Maximal number of replicas.")
-	p.markFlagMakesRevision("max-scale")
-
```

Comment:
Can we keep this, but hidden, for back compat?

Reply:
Do you mean the option "min-scale" and "max-scale"? I moved the functionality for scale-min/scale-max below scale so that it would be alphabetical (prob should have put max first.) The functionality for min-scale, max-scale has been removed.

Reply:
I added them back with MarkHidden